### ISSUE TITLE
p5.Part.removePhrase(name) is not working 

### ISSUE THREAD

**Issue by lmccart** [processing/p5.js-sound#127]
reposting via @xievenlonger (https://github.com/processing/p5.js/issues/1538)

There's an error within p5.Sound that prevents the mentioned function from working. 

```javascript
    p5.Part.prototype.removePhrase = function (name) {
        for (var i in this.phrases) {
            if (this.phrases[i].name === name) {
                this.phrases.split(i, 1); // Should actually be SPLICE - than it works
            }
        }
    };
```

Sorry, I'm not sure how to create pull requests or anything :(

**Comment by rjbultitude** [comment]
I'm happy to fix this @lmccart . I've already forked and have a branch ready for a pull request, so let me know whether someone else is on it or not.

**Comment by therewasaguy** [comment]
@rjbultitude a PR would be great, thank you!

**Comment by therewasaguy** [comment]
fixed with #133
